pheme/transformation/scanreport/gvmd.py

```diff
@@ -259,6 +259,8 @@ def __create_results(
             'nvt.cvss_base',
             'nvt.tags_interpreted',
             'nvt.refs.ref',
+            'nvt.solution.type',
+            'nvt.solution.text',
```

Comment:
I don't think these two lines belong into this pr but nevertheless it is fine

Reply:
right, going to separate.